README.rst

```diff
@@ -65,6 +65,7 @@ Backup scripts / Borg wrappers
 - https://gitlab.com/WoJ/systemd-borg (systemd timer to lauch a backup with a safe autogenerated name and prune existing backups afterwards)
 - https://gitlab.com/lefeverd/borgwrapper (Yet another simple wrapper script, using a YAML configuration file and cron/LaunchAgent to automate backups. Also possible to send a summary by mail)
 - https://hub.docker.com/r/produktion/borg/ (Alpine based docker container)
+- https://github.com/michabbs/proxborg (automates backup of Proxmox vm's (both qemu and lxc) to borg repository)
```

Comment:
(automates backup of Proxmox VMs (both qemu and lxc) to a borg repository)

Reply:
did you see my feedback / can you fix grammar/typos?

Reply:
I copied and pasted directly from their repo.

Reply:
https://github.com/michabbs/proxborg/pull/1